Cargo.toml

```diff
@@ -78,7 +78,7 @@ brotli = "3.3.4"
 bytes = "1.4.0"
 cache_control = "=0.2.0"
 cbc = { version = "=0.1.2", features = ["alloc"] }
-chrono = { version = "0.4", default-features = false, features = ["std", "serde", "clock"] }
+chrono = { version = "0.4", default-features = false, features = ["std", "serde"] }
```

Comment:
Maybe add a comment that people should not add the "clock" feature and outline the reasons why. That will help prevent it being added again.